{{cookiecutter.project_shortname}}/templates/invenio_app_ils/notifications/{{cookiecutter.project_shortname}}_footer.html

```diff
@@ -0,0 +1,25 @@
+{% raw %}
+{% block footer_plain %}
+Check your library loans and requests in your profile page <{{ spa_routes.HOST }}{{ spa_routes.PATHS["profile"] }}>.
+
+Kind regards,
+
+CERN Library
```

Comment:
Let's remove this, and put {{ cookiecutter.project_shortname }} only, nothing below.